infra/resources/prod/storage.tf

```diff
@@ -2,3 +2,67 @@ data "azurerm_storage_account" "immutable_lv_audit_logs_storage" {
   name                = replace("${local.common_project}-lv-logs-im-st", "-", "")
   resource_group_name = data.azurerm_resource_group.core_domain_data_rg.name
 }
+
+###################
+## IOWEB-PROFILE ##
+###################
+resource "azurerm_resource_group" "io_web_profile_fe_rg" {
+  name     = format("%s-%s-%s-%s-ioweb-fe-rg-01", local.prefix, local.env_short, local.location_short, local.domain)
+  location = local.location
+}
+
+module "storage_account_io_web_profile_fe" {
+  source = "github.com/pagopa/dx//infra/modules/azure_storage_account?ref=main"
+
+  // s tier -> Standard LRS
+  // l tier -> Standard ZRS
+  tier = "l"
+
+  # NOTE: domain omitted for characters shortage
+  environment = {
+    prefix          = local.prefix
+    env_short       = local.env_short
+    location        = local.location
+    app_name        = replace("ioweb-profile", "-", "")
+    instance_number = "01"
+  }
+  access_tier = "Hot"
+
+  resource_group_name                  = azurerm_resource_group.io_web_profile_fe_rg.name
+  subnet_pep_id                        = data.azurerm_subnet.private_endpoints_subnet.id
+  private_dns_zone_resource_group_name = data.azurerm_resource_group.rg_common.name
+
+  # storage should be accessible by CDN via private endpoint
+  # see https://learn.microsoft.com/en-us/azure/frontdoor/standard-premium/how-to-enable-private-link-storage-account
+  force_public_network_access_enabled = false
+  subservices_enabled = {
+    blob = true
+    # not used
+    file  = false
+    queue = false
+    table = false
+  }
+  blob_features = {
+    versioning = true
+    change_feed = {
+      enabled = false
+    }
+    immutability_policy = {
+      enabled = false
+    }
```

Comment:
We can safely remove this settings, as `false` is the default value.

Reply:
unfortunately, declaring at least 1 option obligates you to declare `change_feed` and `immutability_policy` objects following the current type definition
```hcl
  type = object({
    restore_policy_days   = optional(number, 0)
    delete_retention_days = optional(number, 0)
    last_access_time      = optional(bool, false)
    versioning            = optional(bool, false)
    change_feed = object({
      enabled           = optional(bool, false)
      retention_in_days = optional(number, 0)
    })
    immutability_policy = object({
      enabled                       = optional(bool, false)
      allow_protected_append_writes = optional(bool, false)
      period_since_creation_in_days = optional(number, 730)
    })
  })

```